src/calls/embedTemplatePage.ts

```diff
@@ -55,6 +59,35 @@ interface CreateIframeOptions {
 
 interface TemplateEmbedConfig {
   features?: TemplateEmbedFeatures | undefined;
+  fonts?: FontSource[] | undefined;
+  appearance?: AppearanceConfig | undefined;
+}
+
+export type FontSource = CssFontSource | CustomFontSource;
+
+export interface CssFontSource {
+  cssSrc: string;
+}
+
+export interface CustomFontSource {
+  family: string;
+  src: string;
+  weight?:
+    | "100"
+    | "200"
+    | "300"
+    | "400"
+    | "500"
+    | "600"
+    | "700"
+    | "800"
+    | "900";
```

Comment:
I'm fine with either, but there is also a one liner for this:
```suggestion
  weight?: `${1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}00`
```